interview-questions.md

```diff
@@ -14,33 +14,33 @@ Researched answer:
 
```

Comment:
You appear to understand the terminology. Do the research and if you see that your answer is correct, find something interesting that you did not know before.